client/package.json

```diff
@@ -3,24 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.12.7",
+    "@babel/preset-env": "^7.12.7",
+    "@storybook/addon-actions": "^6.1.3",
+    "@storybook/addon-essentials": "^6.1.3",
+    "@storybook/addon-links": "^6.1.3",
+    "@storybook/node-logger": "^6.1.3",
+    "@storybook/preset-create-react-app": "^3.1.5",
+    "@storybook/react": "^6.1.3",
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.2.2",
+    "eslint-plugin-react": "^7.21.5",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.1",
     "react-three-fiber": "^5.3.3",
+    "styled-components": "^5.2.1",
```

Comment:
If it is not used yet, remove it :)

Reply:
Thanks for review, I removed styled components already